health-services/project-factory/src/server/utils/microplanIntergration.ts

```
@@ -0,0 +1,969 @@
+    ),
+    facilityBoundaryMap,
+    localizedHeadersMap
+  );
+  logger.info(
+    `workbook updated for facility with the data received from microplan`
+  );
+
+  const responseData =
+    updatedWorksheet && (await createAndUploadFile(workbook, request));
+  logger.info(
+    "facility File updated successfully:" + JSON.stringify(responseData)
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)